3.9/http/database-notes-on-databases.md

```diff
@@ -42,9 +42,16 @@ Database Name
 A single ArangoDB instance can handle multiple databases in parallel. When
 multiple databases are used, each database must be given an unique name.
 This name is used to uniquely identify a database. The default database in
-ArangoDB is named `_system`. The database name is a string consisting of only
-letters, digits and the _ (underscore) and - (dash) characters. User-defined
-database names must always start with a letter. Database names are case-sensitive.
+ArangoDB is named `_system`.
+
+There are two naming conventions available for database names: the **traditional**
+and the **extended** naming conventions. Wether the former or the latter is
+active depends on the `--database.extended-names-databases` startup option.
+The extended naming convention is used if enabled, allowing many special and
+UTF-8 characters in database names. If set to `false` (default), the traditional
+naming convention will be enforced.
```

Comment:
```suggestion
naming convention will be enforced. 
Please note that while it is possible to change the value of the 
`--database.extended-names-databases` from `false` to `true` to enable 
extended names, the reverse is not true. Once the extended names have 
been enabled they will remain permanently enabled so that existing
databases with extended names remain accessible.
In a cluster, it is also required to set the value of the startup option
consistently on all coordinators and DB servers.
```